internal/manifests/manifestutils/constants.go

```diff
@@ -6,4 +6,18 @@ const (
 	// nolint #nosec
 	// BearerTokenFile declares the path for bearer token file for service monitors.
 	BearerTokenFile string = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+
+	ConfigVolumeName = "tempo-conf"
```

Comment:
Please add comments newly added public constants in this file

Reply:
done